pkg/features/kube_features.go

```diff
@@ -1076,21 +1076,21 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	ImageMaximumGCAge: {Default: true, PreRelease: featuregate.Beta},
 
-	InTreePluginAWSUnregister: {Default: false, PreRelease: featuregate.Alpha},
+	InTreePluginAWSUnregister: {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.32
```

Comment:
I don't think we should move a featuregate directly from "Alpha" to "GA".

cc @jsafrane @msau42

Reply:
but the in-tree code was removed, so the beta stage is meaningless.